gitlab/v4/objects/appearance.py

```diff
@@ -39,7 +39,7 @@ def update(
         self,
         id: Optional[Union[str, int]] = None,
         new_data: Optional[Dict[str, Any]] = None,
-        **kwargs: Any
+        **kwargs: Any,
```

Comment:
I didn't really dig into why this is now suddenly an issue, maybe the removal of deps freed up some newer versions of transitive deps that require this change now.